.changeset/swift-phones-cheat.md

```diff
@@ -0,0 +1,12 @@
+---
+'@backstage/cli': minor
```

Comment:
I think we can call this a patch tbh, `.tsx` and `.jsx` wasn't ever really fully supported, and especially after the `swc` switch this is much more of a fix.

Reply:
Aight, done!